### ISSUE TITLE
Newest version of clint (0.3.2) appears to break legit

### ISSUE THREAD

**Issue by suan** [kennethreitz/legit#129]
With a default install on OSX I get the following error when trying to run `legit`:
```shell
$ legit
Traceback (most recent call last):
  File "/usr/local/bin/legit", line 8, in <module>
    load_entry_point('legit==0.1.1', 'console_scripts', 'legit')()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 318, in load_entry_point
    return get_distribution(dist).load_entry_point(group, name)
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 2221, in load_entry_point
    return ep.load()
  File "/System/Library/Frameworks/Python.framework/Versions/2.7/Extras/lib/python/pkg_resources.py", line 1954, in load
    entry = __import__(self.module_name, globals(),globals(), ['__name__'])
  File "/Library/Python/2.7/site-packages/legit/cli.py", line 15, in <module>
    from clint import args
ImportError: cannot import name args
```

However, when I uninstall `clint` and force install `clint` version 0.2.4, the error goes away.

**Comment by the-shank** [comment]
ping @kennethreitz

**Comment by techtonik** [comment]
`legit` is broken on Ubuntu =/

https://bugs.launchpad.net/ubuntu/+source/legit/+bug/1412879

**Comment by blueyed** [comment]
#139 has been merged finally in https://github.com/kennethreitz/legit/commit/869ad2846524c73f335b84f4151d05b09befb73d a few days ago.

I've left a comment on the Ubuntu bug report.

Your best bet is to install it manually from Git.

**Comment by blueyed** [comment]
This issue can be closed.

**Comment by turtlemonvh** [comment]
I can confirm that `pip install legit` was problematic, but `pip install https://github.com/kennethreitz/legit/zipball/master` works fine.  Thanks.

**Comment by jorgebg** [comment]
`pip install legit` didn't work for me neither. I had to install it as @turtlemonvh said :disappointed:

**Comment by kennethreitz** [comment]
the currently recommended installation method is with homebrew. we should fix this. 

Could someone step up to help maintain this?

**Comment by jorgebg** [comment]
Looks like the problem is that the package at pypi is outdated (2012) https://pypi.python.org/pypi/legit
Is the develop branch stable? If it is, just run `python setup.py publish` :smile: 
If it isn't, I can do a pull request with the clint fix to the master branch

**Comment by howardroark** [comment]
@kennethreitz I would be happy to maintain this.  I would really like to see this updated in the "cheese shop" so that I can install it on Linux easily.

**Comment by skorokithakis** [comment]
+1

**Comment by techtonik** [comment]
+1 for "cheese shop" release.

**Comment by hickford** [comment]
Published v0.2.0 to PyPI https://pypi.python.org/pypi/legit/

**Comment by techtonik** [comment]
!m @hickford
